ipaserver/plugins/baseuser.py

```diff
@@ -360,6 +362,13 @@ class baseuser(LDAPObject):
             label=_('Certificate'),
             doc=_('Base-64 encoded user certificate'),
         ),
+        Str(
+            'ipacertmapdata*',
+            cli_name='data',
```

Comment:
The CLI name should be `certmapdata`, `data` is far too generic and disconnected from certificate mapping.